ocastio/src/ocastio/db.clj

```diff
@@ -354,6 +354,15 @@ JOIN org2user ON ballot.org_id = org2user.org_id
 WHERE ballot.org_id = ? AND org2user.user_id = ?
 LIMIT 1" org-id user-id]))))
 
+(defn last-vote [ballot-id user-id]
+  (if (some nil? [ballot-id user-id])
+    false
+    (fvf (jdbc/query db-spec ["
```

Comment:
Simplify this to

    (if (and ballot-id user-id)
      … query …)

As `if` returns `nil` if the else is not provided.